sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -221,26 +226,32 @@ def value(self, new_value: str) -> None:
             temp = json.loads(new_value)
             temp["id"] = self.feature_id
             self._value = json.dumps(temp)
-            self.enabled = temp.get("enabled", None)
+            self.enabled = temp.get("enabled", False)
+            self.display_name = temp.get("display_name", None)
+            self.description = temp.get("description", None)
             self.filters = None
             conditions = temp.get("conditions", None)
             if conditions:
                 self.filters = conditions.get("client_filters", None)
         except (json.JSONDecodeError, ValueError):
             self._value = new_value
-            self.enabled = None
+            self.enabled = False
             self.filters = None
 
     @classmethod
-    def _from_generated(cls, key_value: KeyValue) -> Union["FeatureFlagConfigurationSetting", ConfigurationSetting]:
+    def _from_generated(cls, key_value: KeyValue) -> "FeatureFlagConfigurationSetting":
```

Comment:
Right, the return value could be `None`. Also key_value type should be `Optional[KeyValue]`.

Reply:
There's only one generated API for `delete` could return None KeyValue, so changed to do `None` check in delete operation so that we no longer need to take care the `None` case here.